content/departments/engineering/teams/cody/cody-marketing.md

```diff
@@ -0,0 +1,46 @@
+# Cody Marketing
+
+Please [submit questions and feedback for this page here](https://sourcegraph.slack.com/archives/C04QTJDTH70).
```

Comment:
```suggestion
Please submit questions and feedback for this page at #wg-selling-cody
```

Reply:
This will automatically link to the slack channel and also have the group show up in text on the page